draft-ietf-ppm-dap.md

```diff
@@ -1283,8 +1286,11 @@ subsections.
 
 In general, reports cannot be aggregated until the Collector specifies an
 aggregation parameter. However, in some situations it is possible to begin
-aggregation as soon as reports arrive. For example, Prio3 has just one valid
-aggregation parameter (the empty string).
+aggregation before a collection request arrives. For example, Prio3 has just one
+valid aggregation parameter (the empty string); or, perhaps the relevant
+aggregation parameter can be determined in a deployment-specific way. In these
+cases, aggregation can begin as soon as the Leader is confident that no
```

Comment:
Maybe be explicit about the requirement instead "aggregation can begin as soon as the leader is willing to discard any further incoming reports for this time window."